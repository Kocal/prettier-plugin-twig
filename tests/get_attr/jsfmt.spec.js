run_spec(__dirname, ["twig"]);
