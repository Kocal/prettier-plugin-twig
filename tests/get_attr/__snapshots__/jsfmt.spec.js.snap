// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get_attr.twig 1`] = `
Object {
  Symbol(jest-snapshot-serializer-raw): "====================================options=====================================
parsers: [\\"twig\\"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
{% set foo = foo. bar %}
{% set foo = foo [ bar ] %}
{% set foo = foo.getFoo ( ) %}
{% set foo = foo.getFoo ('foo' ) %}
{% set foo = foo.getFoo ('foo', \\"bar\\", 1123  , [1,2,3,4]
    ) %}

{% set foo = foo.getFoo('aLongLongLongLongLongLongLongLongLongLongLongLongValue', 'aLongLongLongLongLongLongLongLongLongLongLongLongValue', 'aLongLongLongLongLongLongLongLongLongLongLongLongValue'  )  %}

{% set foo = foo.getFoo(     'aLongLongLongLongLongLongLongLongLongLongLongLongValue', ['aLongLongLongLongLongLongLongLongLongLongLongLongValue', 'aLongLongLongLongLongLongLongLongLongLongLongLongValue', ['aLongLongLongLongLongLongLongLongLongLongLongLongValue', {aLongLongLongLongLongLongLongLongLongLongLongLongKey: 'aLongLongLongLongLongLongLongLongLongLongLongLongValue'}]]) %}

{%   set foo = foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo(foo.getFoo())))))))))))))))))))) %}

=====================================output=====================================
{% set foo = foo.bar %}
{% set foo = foo[bar] %}
{% set foo = foo.getFoo %}
{% set foo = foo.getFoo(\\"foo\\") %}
{% set foo = foo.getFoo(\\"foo\\", \\"bar\\", 1123, [1, 2, 3, 4]) %}

{% set foo = foo.getFoo(
    \\"aLongLongLongLongLongLongLongLongLongLongLongLongValue\\",
    \\"aLongLongLongLongLongLongLongLongLongLongLongLongValue\\",
    \\"aLongLongLongLongLongLongLongLongLongLongLongLongValue\\"
) %}

{% set foo = foo.getFoo(
    \\"aLongLongLongLongLongLongLongLongLongLongLongLongValue\\",
    [
        \\"aLongLongLongLongLongLongLongLongLongLongLongLongValue\\",
        \\"aLongLongLongLongLongLongLongLongLongLongLongLongValue\\",
        [
            \\"aLongLongLongLongLongLongLongLongLongLongLongLongValue\\",
            {
                aLongLongLongLongLongLongLongLongLongLongLongLongKey: \\"aLongLongLongLongLongLongLongLongLongLongLongLongValue\\"
            }
        ]
    ]
) %}

{% set foo = foo.getFoo(
    foo.getFoo(
        foo.getFoo(
            foo.getFoo(
                foo.getFoo(
                    foo.getFoo(
                        foo.getFoo(
                            foo.getFoo(
                                foo.getFoo(
                                    foo.getFoo(
                                        foo.getFoo(
                                            foo.getFoo(
                                                foo.getFoo(
                                                    foo.getFoo(
                                                        foo.getFoo(
                                                            foo.getFoo(
                                                                foo.getFoo(
                                                                    foo.getFoo(
                                                                        foo.getFoo(
                                                                            foo.getFoo(
                                                                                foo.getFoo
                                                                            )
                                                                        )
                                                                    )
                                                                )
                                                            )
                                                        )
                                                    )
                                                )
                                            )
                                        )
                                    )
                                )
                            )
                        )
                    )
                )
            )
        )
    )
) %}

================================================================================",
}
`;
